### ISSUE TITLE
inspect: why agent isn't reconnecting

### ISSUE THREAD

**Issue by Eywek** [keymetrics/pm2-io-agent-node#9]
missing applications in keymetrics nomad bucket

**Comment by Eywek** [comment]
under node 10:
```
root@7ae1934cf6c7:~# node_modules/mocha/bin/mocha tests/transport.reconnect.js


  Transporter
    ✓ construct without error
  agent:transport Init new websocket transport with endpoint: ws://localhost:64000 and headers: [X-Test: true] +0ms
    ✓ should set config correctly
  agent:transport Connect transporter to websocket server +8ms
    ✓ should connect correctly (98ms)
  agent:transport Websocket connected +96ms
    ✓ should send message correctly
    ✓ should receive message correctly
  agent:transport Websocket connection is closed, try to reconnect +18ms
  agent:transport Connect transporter to websocket server +0ms
  agent:transport Got an error on websocket connection: socket hang up +12ms
  agent:transport We can't get any response to ping from websocket server, trying to reconnect +977ms
  agent:transport Connect transporter to websocket server +0ms
  agent:transport Got an error on websocket connection: socket hang up +4ms
  agent:transport We can't get any response to ping from websocket server, trying to reconnect +1s
  agent:transport Connect transporter to websocket server +0ms
  agent:transport Got an error on websocket connection: socket hang up +4ms
  agent:transport We can't get any response to ping from websocket server, trying to reconnect +1s
  agent:transport Connect transporter to websocket server +1ms
  agent:transport Got an error on websocket connection: socket hang up +2ms
  agent:transport We can't get any response to ping from websocket server, trying to reconnect +1s
  agent:transport Connect transporter to websocket server +1ms
  agent:transport Got an error on websocket connection: socket hang up +7ms
```

**Comment by Eywek** [comment]
Can't reproduce but will be fixed with 8851a1fabea355ecdaa3621b371592f284a01e0a i think
